data/org.freedesktop.portal.ScreenCast.xml

```diff
@@ -265,6 +265,19 @@
               This property was added in version 3 of this interface.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>mapping_id s</term>
+            <listitem><para>
+              An optional identifier used to map different aspects of the
+              resource this stream corresponds to.
+	    </para><para>
+	      When used in a remote desktop session, the mapping_id can be used to
+              match a libei region of absolute libei device devices. There may
+              be multiple absolute libei devices, and each device may have
+              multiple regions, but a mapping_id will only match one of these
+              regions. This property was added in version 5 of this interface.
```

Comment:
```suggestion
              regions per device. This property was added in version 5 of this interface.
```
Because in theory we could have 2 absolute devices each with a region that maps to stream S1 while stream S2 may only have one (or none).